src/view/element.js

```diff
@@ -713,6 +713,28 @@ export default class Element extends Node {
 			( attributes == '' ? '' : ` ${ attributes }` );
 	}
 
+	/**
+	 * Creates element instance from provided viewElementDefinition.
+	 *
+	 * @param {module:engine/view/viewelementdefinition~ViewElementDefinition} viewElementDefinition
+	 * @returns {module:engine/view/element~Element}
+	 */
+	static createFromDefinition( viewElementDefinition ) {
```

Comment:
Since this method will be removed from the view element I would put it in `definition-based-converters.js`.